apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -353,3 +369,41 @@ const getTrialSystemConfigFromEnvironment: RE.ReaderEither<
     }),
   ),
 );
+
+const getPidIssuerConfigFromEnvironment: RE.ReaderEither<
+  NodeJS.ProcessEnv,
+  Error,
+  PidIssuerApiClientConfig
+> = pipe(
+  sequenceS(RE.Apply)({
+    ipzApiClientCertificate: pipe(
+      readFromEnvironment("PidIssuerApiClientCertificate"),
+      RE.map(decodeBase64String),
+    ),
```

Comment:
The variables should also be renamed in `pidIssuer...`
```suggestion
    pidIssuerApiClientCertificate: pipe(
      readFromEnvironment("PidIssuerApiClientCertificate"),
      RE.map(decodeBase64String),
    ),
```